lhotse/audio.py

```diff
@@ -1331,16 +1334,16 @@ def add_to_mix(
         gain = 1.0
         if snr is not None:
```

Comment:
I would change this to 
```suggestion
        if snr is not None or self.reference_energy == 0:
```